db/migrate/20200820122057_add_proxy_selector_override_to_remote_execution_feature.rb

```diff
@@ -0,0 +1,5 @@
+class AddProxySelectorOverrideToRemoteExecutionFeature < ActiveRecord::Migration[4.2]
+  def change
+    add_column :remote_execution_features, :proxy_selector_override, :string
```

Comment:
Would `proxy_selector` do?
```suggestion
    add_column :remote_execution_features, :proxy_selector, :string
```
Apart of that, lets get it in! :+1:

Reply:
With the `_override` suffix it is clear that if set it overrides some other value and if unset, some "normal" value is used. If we change it to `proxy_selector` only I'd say it stops being clear what no value in this column means

Reply:
Fair enough. I still think blank value = I do not care 'bout my proxy is fair as well, but ok. No strict opinion here.